src/Domain/Creators/DomainCreator.hpp

```diff
@@ -98,6 +102,13 @@ class DomainCreator {
   /// Obtain the initial refinement levels of the blocks.
   virtual std::vector<std::array<size_t, VolumeDim>> initial_refinement_levels()
       const noexcept = 0;
+
+  /// Retrieve the functions of time used for moving meshes.
+  virtual std::unordered_map<
```

Comment:
same comment about `->` for the long return.
Why isn't this one a pure virtual?

Reply:
Good question! Not having it be pure virtual allows a "default implementation" so that not all derived classes need to implement the empty map return, it gets inherited :)